docs/source-fabric/api/utilities.rst

```diff
@@ -0,0 +1,15 @@
+:orphan:
+
+.. include:: ../links.rst
+
+##########################
+lightning.fabric.utilities
+##########################
+
+.. autofunction:: lightning.fabric.utilities.seed.seed_everything
+
+.. autofunction:: lightning.fabric.utilities.seed.pl_worker_init_function
+
+.. autoclass:: lightning.fabric.utilities.types.Optimizable
+
+.. autoclass:: lightning.fabric.utilities.types.Steppable
```

Comment:
I would prefer not to expose these three in the API docs here. They are not meant to be used by the user directly. cc @carmocca

Reply:
ok, then you can't refer them in docs, shall I remove the cross-link?

Reply:
Since pl_worker_init_function is referenced in seed_everything, maybe leave this one. But Optimizable and Steppable can be removed I think.